app/javascript/react/screens/App/common/helpers.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
do we still need these empty "string" helpers?

Reply:
They are being used in only one place, and will be removed when Aparna finishes her stuff for plan wizard step 3. I moved them to helpers temporarily to declutter the PlanWizard code.